### PR TITLE
use table - add history db url

### PR DESCRIPTION
**Pull request by FredericHeem**

### CODE REVIEW

File: client/stellar_postgres.ng.coffee [hunk 1]
```diff
@@ -1,8 +1,9 @@
 data =
   ledgerheaders: new PgSubscription 'lastLedgerHeaders'
+  historyLedgerheaders: new PgSubscription 'historyLastLedgerHeaders'
```
Comment (buhrmi) [inline client/stellar_postgres.ng.coffee]: this publication does not seem to be included in the PR. did you commit everything?

**Comment by buhrmi** [comment]
I've been trying to avoid a dependency on horizon API and/or database. This PR would introduce one, it seems. Can the same information be discovered through the live pg database?

**Comment by FredericHeem** [comment]
The use of horizon is optional, it gives more information about the transactions and the rest of the ledger state.

**Comment by buhrmi** [comment]
I really appreciate getting help with the layout and formatting, but this PR includes some stuff that conflicts with what I'm envisioning for this project:
1. Functionality based on horizon database.
2. Mentioning of XLM. This app should run on any stellar-core based network. The native currency should not be assigned a name.
So I've decided against merging this PR. Feel free to submit a PR that includes layout improvements.
